modules/tensor_mechanics/doc/content/modules/tensor_mechanics/1d_elastic_waves.md

```diff
@@ -1,101 +1,127 @@
 
-Now, we consider the plane wave $u= r e^{i(\omega t - k x)}$ is a solution for [eq3], with $r$ being the complex amplitudes, and $k$ is the wave number and $i=\sqrt{-1}$. As we noted before Young's modulus and the density are constants and equal to unity, thus, $c_p=1$ where $c_p=\sqrt \frac{E}{\rho}$ is the longitudinal wave velocity .
-
-!alert note
-One can verify that $r e^{i(\omega t - k x)}$ is a solution for [eq3]$\Longleftrightarrow$ $k^2= \frac{\omega^2}{c_p^2}$ by direct substitution.
-
-Now, [eq2] and the associated BCs. and can be written as,
+The boundary conditions also need to be converted to the frequency domain through a Fourier transform.  A time harmonic Neumann BC given by
 \begin{equation}
-     -\nabla \cdot \sigma- \omega^2 u = 0 \label{eq4}
+    f= A \, cos(2 \pi f t) \qquad \text{at} \, x=L
+    \label{eq4}
 \end{equation}
-
+with amplitude $A$ and frequency $f$, is transformed to
 \begin{equation}
-    \nabla u= ik \, u \, \qquad \text{at} \, x=0,
-    \\
-    u= A \, cos(2 \pi f t) \, \qquad \text{at} \, x=L,\label{eq5}
+    F= A \, \qquad \text{at} \, x=L
+    \label{eq5}
 \end{equation}
 
-Transforming [eq4] and the associated boundary conditions in [eq5] in time to the frequency domain, we obtain:
-\begin{equation}
-     -\nabla \cdot \hat \sigma- \omega^2  U = 0 \label{eq6}
-\end{equation}
-with the following boundary conditions:
+# Cantilever Beam Example
+
+The cantilever beam shown in [cantilever] is subjected to a time harmonic force on the right side in the out-of-plane and vertical directions.  In this example the frequency of the time varying load is swept over a range.  The displacement at the midpoint of the beam is recorded at each frequency.  This type of output, displacement as a function of frequency, is a frequency response function (FRF) or transfer function.  Displacement peaks in the FRF indicate natural frequencies.  
+
+!media media/tensor_mechanics/Cantilever_beam.png style=width:60%; caption=2D cantilever problem with a prescribed displacement boundary condition on the right end. id=cantilever
+
+The analytic solution for the free vibration of a cantilever [Euler Bernoulli beam](https://en.wikipedia.org/wiki/Euler%E2%80%93Bernoulli_beam_theory).  The analytic eigenvalues, $\omega_n$, are given by
 \begin{equation}
-    \nabla U= ik \, U \, \qquad \text{at} \, x=0,
-    \\
-U= 0.5 \, \qquad \text{at} \, x=L,\label{eq7}
+  \omega_n=k^2_n\sqrt\frac{EI^2}{\rho A L^4}
 \end{equation}
+where $I$ is the moment of inertia, $A$ the cross sectional area, $L$ the beam length, and $k_n$ are the wave numbers.
+For a cantilever beam, the first three wave numbers, $k_n$, are\\
```

Comment:
I think you computed the first three wavenumbers assuming some values for E, \rho which I do not see here in the text.

Reply:
I just copied these values from wikipedia.  I didn't solve the equations.

Reply:
I changed the wording to put the lengths used to compute k_n before the equation.  I use the material properties to compute omega_n from k_n